test/test_common.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+
+# Unit tests for the common module
+class CommonTest < MiniTest::Test
```

Comment:
the test looks great, the only thing I wonder about is whether openshift entities should be tested here and not in their  respective client